### DISCUSSION TITLE
Menu caché

### DISCUSSION THREAD

**Discussion by Mathis554** [witsa/synapps#738] [Q&A]
Bonjour,
En écument les modèles de démo de Synapps j'ai constaté qu'il était possible de faire des menus qui disparaissent quand on clique sur un bouton j'ai voulu le reproduire mais je n'arrive pas

![Capture d’écran 2024-07-01 153936](https://github.com/witsa/synapps/assets/174346872/84badeaf-389a-43be-9688-1dd1ed459dbd)

![Capture d’écran 2024-07-01 153957](https://github.com/witsa/synapps/assets/174346872/0c21a068-e985-441f-bb32-200a288d6c0d)

**Comment by Synfactory** [comment]
hello,
si jamais ça peut aider, ci joint une scene : 
```
SYNAPPS-STUDIO-SCENE|{"config":{"key":"scene1","name":"Scène 1"},"leadActor":{"type":"layout/stack","key":"stack1","properties":{"verticalAlignment":"expand"},"children":[{"type":"layout/stack","key":"stack2","properties":{"height":"2em","width":"10em","horizontalAlignment":"middle"},"children":[{"type":"display/text","key":"text1","properties":{"content":"Invisible","width":"10em","cursor":"pointer","borderColor":"rgba(0, 0, 0, 1)","borderStyle":"solid"}}],"events":{"onClick":["context.getActor('stack3').properties.visible = false;"]}},{"type":"layout/stack","key":"stack4","properties":{"height":"2em","width":"10em","horizontalAlignment":"middle"},"children":[{"type":"display/text","key":"text2","properties":{"content":"Visible","width":"10em","cursor":"pointer","borderColor":"rgba(0, 0, 0, 1)","borderStyle":"solid"}}],"events":{"onClick":["context.getActor('stack3').properties.visible = true;"]}},{"type":"layout/stack","key":"stack3","properties":{"backgroundColor":"rgba(255, 0, 0, 1)","verticalAlignment":"expand","visible":false}}]}}
``` 
j'ai créé trois acteurs : 
- un acteur empilement avec un texte "Invisible"
- un acteur empilement avec un texte "Visible"
- un acteur empilement à cacher/afficher

sur l'acteur "Invisible" tu crées un évènement "onClick" avec le code suivant : 
```javascript
context.getActor('stack3').properties.visible = false;
``` 
'stack3' étant la clé de l'acteur que tu veux rendre invisible

à l'inverse sur l'acteur "Visible" c'est pareil mais avec true.

**Reply by Mathis554** [reply]
Oh bas merci de l'aide 
Le problème c'est que je ne peux pas superposer un acteur sur la zone rouge 😊.
